fdbserver/Status.actor.cpp

```diff
@@ -594,14 +599,12 @@ struct RolesInfo {
 				}
 			}
 
-			if (iface.metadata.present()) {
-				obj["storage_metadata"] = iface.metadata.get().toJSON();
-				// printf("%s\n", metadataObj.getJson().c_str());
-			}
-
 		} catch (Error& e) {
 			if (e.code() != error_code_attribute_not_found)
 				throw e;
+			else {
+				TraceEvent(SevWarnAlways, "StorageServerStatusJson").error(e);
```

Comment:
Can we make SevError in simulation just to find out attributes not found? or Is this common to have attributes not found?

Reply:
I'm thinking the same thing but let's address it in another PR.